src/components/AbstractSelectableComponent.js

```diff
@@ -158,7 +143,7 @@ export default class AbstractSelectableComponent extends React.Component {
           {this.renderLayersMenu()}
         </div>
         <DeckGL ref={this.deckRef} {...deckProps}>
-          {this.renderImagesFromView}
+          {this.initializeViewInfo}
         </DeckGL>
```

Comment:
we need to initialize the view info for the pubsub - this is why the tooltip was not working